packages/astro/src/core/ssr/index.ts

```diff
@@ -389,7 +286,8 @@ export async function render(renderers: Renderer[], mod: ComponentInstance, ssrO
 	// run transformIndexHtml() in dev to run Vite dev transformations
 	if (mode === 'development') {
 		const relativeURL = filePath.href.replace(astroConfig.projectRoot.href, '/');
-		html = await viteServer.transformIndexHtml(relativeURL, html, pathname);
+		console.log('TRANFORM', relativeURL, html);
```

Comment:
whoops. I think this still can't be removed for non static builds, so this should be updated.